### DISCUSSION TITLE
Automatically clean _output before each run

### DISCUSSION THREAD

**Discussion by magroski** [Codeception/Codeception#6204] [Ideas]
I know that there is a command `codeception clean` that clears the _output folder, but I think that would be nice to have the option to always clear the folder before running the tests. Maybe a flag, maybe a config in codeception.yml

**Comment by DavertMik** [comment]
Please try this extension https://github.com/oprudkyi/codeception-events-scripting
And add `codecept clean` before all tests

**Comment by Luc45** [comment]
It would be nice if this could be built-in :)
